packages/components/pill/README.mdx

```diff
@@ -60,4 +64,9 @@ Example of the Pill component with dragHandle, in that case Icon.
 
 ## Accessibility
 
-- Label value is passed in the component as a aria-label and title in the span element, like follows `aria-label={label} title={label}`.
+- Label value is passed in the component as a title in the span element, like follows `title={label}`.
+- The default drag component has a `aria-label="Drag handler"`, custom components should also include an `aria-label`
```

Comment:
It is not 100% clear to me what we mean by custom components. I think an example here could be nice.